src/fragments/lib-v1/auth/native_common/signout/common.mdx

```diff
@@ -16,11 +16,11 @@ Calling signOut without any options will just delete the local cache and keychai
 
 [Amazon Cognito now supports token revocation](https://aws.amazon.com/about-aws/whats-new/2021/06/amazon-cognito-now-supports-targeted-sign-out-through-refresh-token-revocation/) and the latest Amplify version will revoke Amazon Cognito tokens if the application is online. This means that the Cognito refresh token cannot be used anymore to generate new Access and Id Tokens.
 
-Access and Id Tokens are short-lived (60 minutes by default but can be set from 5 minutes to 1 day). After revocation, these tokens cannot be used with Cognito User Pools anymore. However, they are still valid when used with other services like AppSync or API Gateway.
+Access and Id Tokens are short-lived (60 minutes by default but can be set from 5 minutes to 1 day). After revocation, these tokens cannot be used with Cognito user pool anymore. However, they are still valid when used with other services like AppSync or API Gateway.
```

Comment:
I appreciate you fixing this, but since this is the only change to this file now (previously I think there were other changes), I would suggest reverting this since it is really not in scope.

There are ~500 references to either "User Pool" or "User Pools" throughout the docs. There will need to be quite a bit of work done to get them all updated.